source/oauth/settings.d

```diff
@@ -387,22 +432,8 @@ class OAuthSettings
     struct LoginData
     {
         SysTime timestamp;
-        ulong   randomId;
         string  scopes;
         bool    redirectUriRequired;
-
-        auto key() @property const nothrow @safe
-        {
-            import std.digest.crc : crc32Of;
-            import std.digest.sha : sha256Of;
-
-            ubyte[20] data;
-            ulong[] data64 = cast(ulong[])(data[0 .. 16]);
-            data64[0] = this.timestamp.toUnixTime;
-            data64[1] = this.randomId;
-            data[16 .. 20] = crc32Of(this.scopes);
-            return sha256Of(data[]);
-        }
+        ubyte[16] key;
```

Comment:
By this change, LoginData.key loses its dependency on `timestamp` and `scopes`, which had a purpose.